### PR TITLE
add test for null results in tree panel (AST-84362)

### PR DESCRIPTION
**Pull request by elchananarb**
By submitting a PR to this repository, you agree to the terms within the [Checkmarx Code of Conduct](../docs/code_of_conduct.md). Please see the [contributing guidelines](../docs/contributing.md) for how to create and submit a high-quality PR for this repo.

### Description

> Describe the purpose of this PR along with any background information and the impacts of the proposed change.

### References

> Include supporting link to GitHub Issue/PR number

### Testing

> Describe how this change was tested. Be specific about anything not tested and reasons why. If this solution has unit and/or integration testing, tests should be added for new functionality and existing tests should complete without errors.
>
> Please include any manual steps for testing end-to-end or functionality not covered by unit/integration tests.

### Checklist

- [ ] I have added documentation for new/changed functionality in this PR (if applicable).
- [ ] All active GitHub checks for tests, formatting, and security are passing
- [ ] The correct base branch is being used

### CODE REVIEW

File: UITests/ComboboxTest.cs [hunk 1]
```diff
@@ -133,5 +133,104 @@ public async Task ProjectsComboboxSearchAsync()
...
+            
+            Assert.IsTrue(scanFound, "Scan 28d29a61-bc5e-4f5a-9fdd-e18c5a10c05b not found");
+            
+            // 4. Check that no Object reference error is displayed
+            var treeViewResults = TestUtils.GetElementByAutomationIdWithNotNullCheck(_checkmarxWindow, "TreeViewResults", "Tree view results not found in Checkmarx window");
+            var results = treeViewResults.FindAllDescendants();
+            
+            foreach (var item in results)
+            {
+                Assert.IsFalse(item.Name.Contains("Object reference not set"), "Error message found: Object reference not set");
```
Comment (greensd4) [inline UITests/ComboboxTest.cs:218]: can we check expected output? not what we "dont expect" to get

Comment (elchananarb) [reply UITests/ComboboxTest.cs:218]: In this test, we do both:
In step 5, we check that the "No results" message appears (expected output
In step 4, we just ensure no internal error is shown

**Review by greensd4 — APPROVED** [review]
